src/components/template/editor/Editor.tsx

```
@@ -0,0 +1,221 @@
+lowlight.register("css", css)
+lowlight.register("js", js)
+lowlight.register("ts", ts)
+
+const CustomDocument = Document.extend({
+    content: "heading block*",
+})
+
+const Editor = (props: IEditorProps): ReactElement => {
+    const { title, contents, thumbnail, onChangeTitle, onChangeContents, onChangeThumbnail } = props
+
+    const uploadToGoogleDrive = (file: File): void => {
```

ESLint

[error] 'uploadToGoogleDrive' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)